docs/data-management/post-session.md

```diff
@@ -374,11 +374,61 @@ ses-01
 
     - [ ] Add stockage horus as an SSH remote.
+    - [ ] The preprocessing will be executed on an HPC. If datalad is not installed on the HPC, the most convenient user-based installation can be achieved by using Conda. 
+
+        ??? tip "Execution within High Performance Computing (HPC)"
+        
+            HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing job scripts.
+            Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
+            Interactive sessions are a great alternative when available.
+            For example, in the case of systems operating SLURM, the following command would open a new interactive session:
+            ``` bash
+            srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
+            ```
+
+    ??? tip "Installing DataLad with Conda"
+    
+        - [ ] Install some conda distribution (in this example we show *Miniconda*):
+            ``` bash
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh
+            ````
+        - [ ] Install DataLad:
+            ``` bash
+            conda install -c conda-forge -y "datalad>=0.19.0"
+            conda install -c conda-forge datalad-container
+            ```
+        - [ ] Next, don't forget to configure the Git identity.
+            ``` bash
+            git config --global --add user.name "Bob McBobFace"
+            git config --global --add user.email bob@example.com
+            ```
+        
+    - [ ] Go in the repository where you usually store datasets and create an empty DataLad dataset by running:
+        ```
+        cd /data/datasets/
+        datalad create -c yoda "hcph"
+        ```
+        Using the `yoda` procedure builds the infrastructure of your dataset so it is compliant to the [Yoda principles](https://handbook.datalad.org/en/latest/basics/101-127-yoda.html).
+
+- [ ] Move the data into the DataLad dataset and save the files in the dataset history (thereby starting the version-control mechanism) using the command below:
+    ``` bash
+    datalad save -m "add: first batch of currently existing data"
+    datalad push
+    ```
+- [ ] As new sessions are run, move the data into their corresponding location in the dataset structure and save the data (remember to replace `<session_id>` in the command line):
+    ``` bash
+    datalad save -m "add: session <session_id>"
+    datalad push
+    ```
+
+## References
+[1]: Hanke, Michael, Yaroslav O. Halchenko, Benjamin Poldrack, Kyle Meyer, Debanjum Singh Solanky, Gergana Alteva, Jason Gors, et al. “Datalad/Datalad: ## 0.14.0 (February 02, 2021).” Zenodo, February 2, 2021. <https://doi.org/10.5281/zenodo.4495661.>
```

Comment:
Following the DOI link leads to a "DOI Not Found" error - maybe we could cite the datalad paper ? Or the idea would be to cite the "frozen" version of datalad that will be used ?

Reply:
Oh thanks for catching this! I pushed a new commit to cite the paper instead (and I checked the DOI link worked this time ;) )